subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/BuildScriptBuilder.java

```diff
@@ -195,11 +228,23 @@ public void generate() {
 
     private static class ConfigSpec {
         private final String comment;
-        private final List<String> configLines;
+        private final Map<BuildInitBuildScriptDsl, List<String>> configLines;
 
-        public ConfigSpec(String comment, List<String> configLines) {
+        ConfigSpec(String comment, String config) {
             this.comment = comment;
-            this.configLines = configLines;
+            this.configLines = new HashMap<BuildInitBuildScriptDsl, List<String>>();
+            List<String> lines = Splitter.on("\n").splitToList(config);
```

Comment:
System specific newline? Should this use the OS newline? Does it matter??

Reply:
This is an internal type and the incoming strings are internal java strings only.

On a related note, all files generated by `init` use `\n` as a line separator, including on Windows. This PR doesn't change that.